src/math.cpp

```diff
@@ -19,4 +27,95 @@ auto abs(const CallContext& ctx) -> Value {
         throw std::runtime_error("bad argument #1 to abs (number expected, got " + x.type() + ")");
     }
 }
+
+auto acos(const CallContext& ctx) -> Value {
+    auto x = ctx.arguments().get(0);
+    auto res = minilua::to_number(ctx);
+
+    if (res != Nil()) {
+        auto num = get<Number>(res);
+
+        if (num.value >= -1 && num.value <= 1) {
+            return Value(std::acos(num.value));
+        } else {
+            return Value("nan");
```

Comment:
Are the nan values returned in math functions in lua actually strings or are they float nans?

Reply:
when i execute it in lua there only stands nan. i dont know if thats a string or a float

Reply:
```
> type(math.acos(3))
number
```